gc/stats/LargeObjectAllocateStats.cpp

```diff
@@ -53,6 +53,35 @@ MM_LargeObjectAllocateStats::newInstance(MM_EnvironmentBase *env, uint16_t maxAl
 	return largeObjectAllocateStats;
 }
 
+void
+MM_LargeObjectAllocateStats::initializeFreememoryProfileMaxSizeClasses(MM_EnvironmentBase *env, uintptr_t veryLargeObjectThreshold, float sizeClassRatio, uintptr_t maxHeapSize)
```

Comment:
since field name (freeMemoryProfileMaxSizeClasses) has upper case 'M',  so should the initialize method name have